### ISSUE TITLE
Alerts list cannot be filtered

### ISSUE THREAD

**Issue by Gnafu** [geosolutions-it/decat_geonode#284]
In the Early Warning phse, the Event Operator can see all the Alerts.
The alerts list cannot be filtered using the "FIlter" options.

For example, select "1hour" in the time range filter and you will still see all the alerts.

Select only a single Alert Level and you will still see all the alerts levels.

**Comment by kappu72** [comment]
I guess the client generates a correct request, for instance http://decat-dev.geo-solutions.it/decat/api/alerts/?page=1&page_size=1000&promoted=true&archived=false&hazard_type__in=wildfire&level__in=watch,information,advisory,warning&regions__code__in=AFG ,
 but in the backend response the alerts list isn't properly filtered. Is the [rest api](https://github.com/geosolutions-it/decat_geonode/wiki/REST-API-Examples#getting-alerts-list-with-filter) changed ?

**Comment by simboss** [comment]
thx for this kappu, I will talk to @cezio and @afabiani about this.

**Comment by cezio** [comment]
Could you verify on dev?

**Comment by Gnafu** [comment]
The filter seems to work, but is there a limit on the visible events number? I noticed that if I deselect the filter I see "all" the events but a specific filter does not display a subset of them.

https://www.screencast.com/t/Hv0Oee4k0kj

**Comment by cezio** [comment]
yep, page size probably (i see in logs that client is querying for 10 items per page):

this will return 3 oilspill: http://decat-dev.geo-solutions.it/decat/api/alerts/?page=1&page_size=10&promoted=true&archived=false&level__in=watch,information,advisory,warning

this will return 4 oilspills: http://decat-dev.geo-solutions.it/decat/api/alerts/?page=1&page_size=20&promoted=true&archived=false&level__in=watch,information,advisory,warning
